Gemfile

```diff
@@ -16,8 +16,8 @@ gem 'pg', '~> 1.1'
 gem 'puma', '~> 5.0'
 
 # Use JavaScript with ESM import maps [https://github.com/rails/importmap-rails]
+gem 'bullet'
```

Comment:
* As stated in the requirements, you are not to use the bullet gem for solving N+1 problems, kindly remove the gem, solve the N+1 issues and add a screenshot of your result to your PR 
![image](https://github.com/gsmalik030/myblog-app/assets/52497246/38c3a72d-31c9-4600-8b49-21890defef6a)